tests/test_iter.py

```diff
@@ -6,14 +6,13 @@
 from hypothesis import (
     given,
     settings,
-    strategies,
 )
 
 from trie import HexaryTrie
 from trie.exceptions import MissingTraversalNode
 from trie.iter import NodeIterator
 from trie.utils.nodes import is_extension_node
-from .utils import make_random_trie
+from .utils import random_trie_strategy
```

Comment:
@carver this is the type of thing that I would typically relegate to `trie.tools.strategies` or something to prevent imports from the `./tests` directory.